velox/docs/functions/iceberg/coverage.rst

```diff
@@ -0,0 +1,26 @@
+=================
+Function Coverage
```

Comment:
Let's add "Iceberg": Iceberg Function Coverage.

Is there official Iceberg function list somewhere? Can we add a link here for reference?